runtime/util/hshelp.c

```diff
@@ -1701,6 +1704,75 @@ fixJNIRefs(J9VMThread * currentThread, J9HashTable * classPairs, BOOLEAN fastHCR
 }
 
 
+typedef struct J9ThreadHashPair {
+	J9VMThread *currentThread;
+	void *userData;
+} J9ThreadHashPair;
```

Comment:
Is this the correct way to pass in the thread and hash table? This is probably not the best place to put it.